index.html

```diff
@@ -346,22 +347,35 @@ <h4>Dubbing Examples</h4>
     <section id="conventions">
       <h2>Documentation Conventions</h2>
       <p>
-        This document uses the same conventions as [[ttml2]] for the specification of parameter attributes, styling attributes and metadata elements. In particular:
-      </p>
-      <p>
-        Section 2.3 of [[ttml2]] specifies conventions used in the [[XML]] representation of elements; and
-        Sections 6.2 and 8.2 of [[ttml2]] specify conventions used when specifying the syntax of attribute values.
-      </p>
-      <p>
-        This specification uses <dfn data-cite="ttml2#terms-feature">Feature</dfn> designations as defined in Appendices E at [[ttml2]]
-        and defines <dfn data-cite="ttml2#terms-extension">Extension</dfn> designations:
-        when making reference to content conformance,
-        these designations refer to the syntactic expression or the semantic capability associated with each designated <a>Feature</a> or <a>Extension</a>;
-        and when making reference to <dfn data-cite="ttml2#terms-processor">Processor</dfn> conformance,
-        these designations refer to processing requirements associated with each designated <a>Feature</a> or <a>Extension</a>.
-        If the name of an element referenced in this specification is not namespace qualified,
-        then the TT namespace applies (see <a href="#namespaces">9.3 Namespaces</a>.)
+        This document uses the following conventions:
       </p>
+      <ul>
+        <li>When refering to an [[XML]] element in the prose, angled brackets and a specific style are used as follows: <code>&lt;someElement&gt;</code>. If the name of an element referenced in this specification
```

Comment:
Aargh I forgot to actually fix this typo in my suggestion, sorry!

```suggestion
        <li>When referring to an [[XML]] element in the prose, angled brackets and a specific style are used as follows: <code>&lt;someElement&gt;</code>. If the name of an element referenced in this specification
```